package.json

```diff
@@ -42,7 +42,7 @@
     "dev:stop": "(cd packages/operation && yarn run dev:stop)",
     "docker": "yarn docker:artifacts && yarn docker:deployer && yarn docker:node",
     "docker:artifacts": "docker rmi --force api3/artifacts && docker build --tag api3/artifacts:latest --file docker/Dockerfile .",
-    "docker:deployer": "docker build --tag api3/deployer:latest --file packages/deployer/docker/Dockerfile .",
+    "docker:deployer": "docker build --tag api3/airnode-deployer:latest --file packages/deployer/docker/Dockerfile .",
```

Comment:
I've changed it, because that the name of the deployer image on dockerhub https://hub.docker.com/r/api3/airnode-deployer and I want the usage to be same.